example/lib/utils/constants.dart

```diff
@@ -1,4 +1,4 @@
 class Constants {
-  static const String apiKey = '3e093592-3e0e-4a52-9601-ead49f794586';
+  static const String devApiKey = '3e093592-3e0e-4a52-9601-ead49f794586';
```

Comment:
I think apiKey is a better name, user will receive prod api key from email and paste here. Using the devApiKey looks confused